xoinvader/ingame.py

```diff
@@ -108,16 +176,16 @@ def __init__(self, owner):
         self.add(self._actor)
         self._events = InGameEventHandler(self)
         self.add(self._create_gui())
-        self.enemy = GenericXEnemy(
-            Point(x=15, y=3), Settings.layout.field.edge)
 
-        self.add(self.enemy)
+        self.wave = TestWave(self.add)
+        self.wave.speed = 1
```

Comment:
So you define `__init__(method, speed=0, start=False)` and just create object without passing unneeded parameters.